content/en/project/Parkinson-fMRI/index.md

```diff
@@ -0,0 +1,162 @@
+github_repo: https://github.com/brainhack-school2023/Moqadam_project
+
+# If you are working on a project that has website, indicate the full url including "https://" below or leave it empty.
+website:
+
+# List +- 4 keywords that best describe your project within []. Note that the project summary also involves a number of key words. Those are listed on top of the [github repository](https://github.com/brainhack-school2020/project_template), click `manage topics`.
+# Please only lowercase letters
+tags: [parkinson's, rsfmri, fmriprep, python, nilearn]
+
+# Summarize your project in < ~75 words. This description will appear at the top of your page and on the list page with other projects..
+
+summary: "This project aims to investigate the functional connectivity patterns in individuals with Parkinson's disease and neuropsychiatric symptoms (PD+NPS) compared to those without the NPS. The study utilizes resting-state fMRI data to analyze the connectivity matrix and identify alterations in functional brain networks associated with PD+NPS. The project also involves familiarizing with data science packages, interpreting neuroimaging data, and advanced visualization techniques. The findings may contribute to understanding the neural mechanisms underlying PD+NPS and inform future research and interventions. The project involves BIDS validation, fMRI preprocessing, functional connectivity analysis, group comparisons, and prediction modeling for mild cognitive impairment."
+
+# If you want to add a cover image (listpage and image in the right), add it to your directory and indicate the name
+# below with the extension.
+image: "![View](https://github.com/brainhack-school2023/Moqadam_project/raw/main/outputs/view.png)"
+---
+<!-- This is an html comment and this won't appear in the rendered page. You are now editing the "content" area, the core of your description. Everything that you can do in markdown is allowed below. We added a couple of comments to guide your through documenting your progress. -->
+
+## Project definition
+
+### Background
+
+
+Inspired by the https://alz-journals.onlinelibrary.wiley.com/doi/full/10.1002/trc2.12371, this project has defined which aims to investigate the functional connectivity patterns in individuals with Parkinson's disease and neuropsychiatric symptoms (PD+NPS) compared to those without the NPS. The study utilizes resting-state fMRI data to analyze the connectivity matrix and identify alterations in functional brain networks associated with PD+NPS. The project also involves familiarizing with data science packages, interpreting neuroimaging data, and advanced visualization techniques. The findings may contribute to understanding the neural mechanisms underlying PD+NPS and inform future research and interventions. The project involves BIDS validation, fMRI preprocessing, functional connectivity analysis, group comparisons, and prediction modeling for mild cognitive impairment.
+
+<<<<<<< HEAD
```

Comment:
This line (up to l40) need to be removed. They come from a merge conflict which was not properly resolved.